### ISSUE TITLE
Full screen map - cumulative mode for DonutMarker

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/web-components#421]
Add a new prop `cumulative?: boolean` to `DonutMarker`

When true, when given label/value pairs  `filtered=85, total=100`, would display a 85% pink and 15% grey donut marker. Mouse-overs would show the correct numbers: 85, and 100.

Do we assume the values are sorted already? Probably yes. This is because when provided with `filtered=100, total=100` we want the `filtered` "segment" of the donut to come first/be coloured and leave the `total` segment invisible/size zero.

So in summary, when `cumulative == true`, assume incoming values are in order, the final value is the total, and any values before that are fractions of that.
